packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -130,7 +130,7 @@ export class ConfigUtil {
 
     const info = await StateAggregator.getInstance();
     const username = defaultUsernameOrAlias
-      ? info.aliases.getUsername(String(defaultUsernameOrAlias))
+      ? info.aliases.resolveValue(String(defaultUsernameOrAlias))
```

Comment:
getUsername returns null when this method is passed the username.  resolveValue returns the passed in string (the username) instead of returning null
<img width="500" alt="2022-10-03_22-56-06" src="https://user-images.githubusercontent.com/46458081/193748628-f7c00258-5ac7-4464-91ac-095e93cdf1c5.png">